crates/gpui/src/text_system/line_wrapper.rs

```diff
@@ -90,6 +98,13 @@ impl LineWrapper {
         })
     }
 
+    pub(crate) fn is_word_char(c: char) -> bool {
+        c.is_ascii_alphanumeric() ||
```

Comment:
Wouldn't this cause problems in languages that follow an english-like syntax but don't contain ASCII characters, such as Spanish with `ñ` or french with `â`?

Reply:
Yes, according to the current logic, non-Ascii may break lines based on characters.

<img width="583" alt="image" src="https://github.com/zed-industries/zed/assets/5518/e378495d-92fb-43d4-b261-1847003688b9">